modules/fts/pages/fts-monitor.adoc

```diff
@@ -0,0 +1,144 @@
+Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+ 
+For example, 
+
+----
+
+curl -XGET -H "Content-Type: application/json" -uAdministrator:asdasd 'http://localhost:9200/api/query/index/<indexName>?longerThan=1ms
+
+Sample Output
+{ "status": "ok", "totalActiveQueryCount": 3, "filteredActiveQueries": { "indexName": "DemoIndex", "longerThan": "1s", "queryCount": 3, "queryMap": { "4": { "QueryContext": { "query": { "query": "ipa" }, "size": 10, "
+from
+   ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "17.340715297s" }, "5": { "QueryContext": { "query": { "query": "german" }, "size": 10, "
+   from
+      ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "9.561917571s" }, "6": { "QueryContext": { "query": { "query": "pale ale" }, "size": 10, "
+      from
+         ": 0, "timeout": 10000, "index": "DemoIndex" }, "executionTime": "1.239720897s" } 
+        } 
+    } 
+}
+
+----
+
+== Monitor Search Index Statistics
+
+Couchbase provides detailed statistics about the full text search indexes. The statistics include various metrics that explain the performance of the search query based on the selected full text search index.
+
+Hold the mouse pointer on each chart to view the statistical details corresponding to the chart. Additionally, click the chart to expand the chart in full view.
+
+Couchbase provides statistics for the full text search indexes on the Dashboard page and the Search page.
+
+* Dashboard page
+* Search page
+
+=== Dashboard page
+
+To view the statistics on the Dashboard page:
+
+1. In the left pane, click *Dashboard*.
+2. On the Dashboard page, click the *Search indexes* drop-down list and select the full text search index you want to view the statistics.
+
+A graphical display of the statistics is as shown below:
+
+[#fts_fts_monitoring_statistics1]
+image::fts-monitoring statistics1.png[,820,align=left]
+
+The statistics display the following graphical charts for full text search index:
+* Search Mutations Remaining: The no. of mutations not yet indexed.
+
+=== Search page
+
+To view the statistics on the Search page:
+
+1. In the left pane, click *Search*.
+2. On the Full Text Indexes page, click the full text search *index name* in the Index Name column.
+3. Click the *Search Index Stats* drop-down arrow to display the statistics.
+4. (Optional) Click the *interval* drop-down list and select the time interval to view the statistics. By default, you can view the statistics for every minute.
+
+A graphical display of the statistics is shown below:
+
+[#fts_fts_monitoring_statistics2]
+image::fts-monitoring statistics1.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query.
+* Search Docs: The no. of documents scanned for the search query.
+* Search Disk Size: The total disk file size occupied by the fts index
```

Comment:
The total size of disk space occupied by the fts index

Reply:
Change no. to "number of" in all occurrences

The total size of disk space occupied by the full text search index

Reply:
```suggestion
* Search Disk Size: The total size of disk space occupied by the fts index.
```